R/find_dependencies.R

```diff
@@ -0,0 +1,18 @@
+find_dependencies <- function(paths = pacta_projects()) {
```

Comment:
I didn't document this function because I forgot; but I forgot because it's mostly 
a helper of another function that I'll submit in another PR: `r_packages()`.

To learn what to expect, please see the titles of each test below.